psi4/src/psi4/cc/ccdensity/kinetic.cc

```diff
@@ -117,6 +117,10 @@ void kinetic(std::shared_ptr<Wavefunction> wfn) {
     outfile->Printf("\t-V/T (corr)            = %20.15f\n", -vcorr / tcorr);
     outfile->Printf("\t-V/T (total)           = %20.15f\n", -vtot / ttot);
 
+    wfn.set_scalar_variable("CC DELTA KINETIC ENERGY", tcorr);
+    wfn.set_scalar_variable("CC DELTA POTENTIAL ENERGY", vcorr);
+    wfn.set_scalar_variable("CC DELTA VIRIAL RATIO", -vtot / ttot + vref / tref);
```

Comment:
What is the "delta" between? Looking at the `tcorr` etc. to be stored I might have named them `CC CORRELATION KINETIC ENERGY`, `CC CORRELATION POTENTIAL ENERGY`, `CC CORRELATION VIRIAL RATIO`.

Reply:
My one concern is that CC CORRELATION RATIO might _look like it means_ "correlation potential" / "correlation kinetic." Using "delta" instead doesn't have that ambiguity. If you aren't worried about that, as long as the psivars are defined clearly in the glossary, I'm happy to change "DELTA" to "CORRELATION."

Reply:
I agree CORRELATION is ambiguous. In my initial reading I would call DELTA vague. I'm not sure which is worse. For consistency I'd probably choose CORRELATION accompanied by clear glossary definitions, but I'm not adamant. There's also CORRECTION that's often used for parts that aren't a full tot - ref, if that seems much better to you.

Reply:
After reviewing Psi usage, "CORRECTION" either appears in contexts that are too dissimilar or that should clearly be "CORRELATION." So "CORRECTION" it is.

Reply:
Where are you seeing CORRECTIONs that should be CORRELATIONs? My intent was `CORL = TOT – REF`, and if a piece didn't meet that, it became a CORRECTION. (Even pieces like `(T) = CCSD(T) – CCSD` are CORRECTION, though that's certainly the perturbative correction to the _correlation_ energy.)

Reply:
With that clarification, I retract my previous statement: Psi usage is correct, and I'll move my keyword to "CORRELATION."